ResidentsSocialCarePlatformApi/V1/Controllers/ResidentsController.cs

```diff
@@ -114,5 +117,28 @@ public IActionResult GetVisitInformation(long personId)
 
             return Ok(visitInformation);
         }
+
+        /// /// <summary>
+        /// Get relationships for a person
+        /// </summary>
+        /// <response code="200">Success. Returns relationships for a person</response>
+        /// <response code="400">Invalid person ID</response>
+        [ProducesResponseType(typeof(Boundary.Responses.Relationships), StatusCodes.Status200OK)]
+        [HttpGet]
+        [Route("{personId}/relationships")]
+        public IActionResult GetRelationships([FromQuery] GetRelationshipsRequest request)
```

Comment:
There might be a little confusion over using query parameters and path parameters here.
If you'd like I'd be happy to jump on a call to talk about the differences and how asp.net implements each one.